WooCommerce/Classes/ViewRelated/NotificationsBadgeController.swift

```diff
@@ -0,0 +1,102 @@
+import Foundation
+import UIKit
+
+final class NotificationsBadgeController {
+    /// Displays or Hides the Dot, depending on the new Badge Value
+    ///
+    func badgeCountWasUpdated(newValue: Int, in tabBar: UITabBar) {
+        guard newValue > 0 else {
+            hideDotOn(.notifications, in: tabBar)
+            return
+        }
+
+        showDotOn(.notifications, in: tabBar)
+    }
+
+    /// Shows the dot in the specified WooTab
+    ///
+    func showDotOn(_ tab: WooTab, in tabBar: UITabBar) {
+        hideDotOn(tab, in: tabBar)
+        let dot = GreenDotView(frame: CGRect(x: DotConstants.xOffset,
```

Comment:
Can this be renamed? `GreenDotView` is misleading

Reply:
Do you have any suggestion? It is actually a green dot at the moment (and it's how it was named before)

Reply:
Maybe `NotificationDotView` or `TakeActionDotView` or `IndicatorDotView` ?